lib/pavilion/plugins/commands/run.py

```diff
@@ -196,11 +204,21 @@ def _get_tests(self, pav_cfg, args, mb_tracker, build_only=False,
         :rtype: []
         """
 
+        repeat_re = re.compile(r'(.*)\*(\d+)')
+
+        tests = []
+        for test_name in args.tests:
+            match = re.search(repeat_re, test_name)
+            if match:
+                name, count = match.groups()
+                tests.extend([name]*int(count))
+            else:
+                tests.append(test_name)
```

Comment:
Make this work in both direction.  
 - 5*test
 - test*5

Reply:
You can drop the re entirely. 
Just check that '*' in test_name, and then do a single split(test_name, '*', 1).

Reply:
You also need to add this to the command argument documentation.